bin/pg_repack.c

```diff
@@ -1322,7 +1325,11 @@ repack_one_table(repack_table *table, const char *orderby)
 	{
 		const char *indexdef;
 		indexdef = getstr(indexres, j, 0);
-		if (error_on_invalid_index) {
+
+		/* Enforce --error-on-invalid-index in all cases unless
+		 * --no_error_on_invalid_index passed by itself
+		 */
+		if (error_on_invalid_index || !no_error_on_invalid_index) {
```

Comment:
Thanks for the PR! What if we completely ignore `error_on_invalid_index` here and make it no-op as it was suggested previously https://github.com/reorg/pg_repack/issues/422#issuecomment-2402913236, but keep it as a valid option and document it as a no-op in the documentation? WDYT?

Reply:
Needs to stay as an option anyway. Can be marked as deprecated, and in 2-3 years can be removed.

Reply:
Yeah, I think this makes sense. Released an updated version, which makes --error-on-invalid-index no-op and marked it as deprecated in helper and docs.

Reply:
Thanks! I'd merge the PR if no objections.